pkg/acceptance/cluster/localcluster.go

```diff
@@ -651,9 +653,15 @@ func (l *LocalCluster) Start() {
 
 	l.monitorCtx, l.monitorCtxCancelFunc = context.WithCancel(context.Background())
 	go l.monitor()
+	var wg sync.WaitGroup
+	wg.Add(len(l.Nodes))
 	for _, node := range l.Nodes {
-		l.startNode(node)
+		go func(node *testNode) {
```

Comment:
Probably want to sprinkle some commentary on `startNode` and `createRoach` that they might be called in parallel. I did a cursory review and nothing looks obviously racy.